src/lib/npm.js

```diff
@@ -42,14 +41,31 @@ async function getNpmToken({npm, options}) {
   log.info('Successfully created npm token.');
 }
 
+function getRegistry(pkg, conf) {
```

Comment:
Where do we use that in semantic-release to retrieve the registry?

AFAK we use https://github.com/semantic-release/npm/blob/master/lib/get-registry.js and its the only place where we read the registry

Reply:
That's super old, it's not in the code base anymore.
We should use https://github.com/semantic-release/npm/blob/master/lib/get-registry.js for consistency.

But that can be done in a different PR. Let me know if you prefer to do it here, otherwise I can merge this PR

Reply:
There's no runtime dep on semantic-release yet. Is there any way to reuse this, without copying?

Reply:
It's in a plugin so not easily.
It's only 2 lines of code and it uses [registry-auth-token](https://github.com/rexxars/registry-auth-token), so I guess it ok.